client/signup/flow-progress-indicator/index.jsx

```diff
@@ -4,10 +4,12 @@
 import React from 'react';
 import { localize } from 'i18n-calypso';
 
-const FlowProgressIndicator = ( { flowLength, positionInFlow, translate } ) => {
+const FlowProgressIndicator = ( { flowLength, positionInFlow, translate, flowName } ) => {
 	if ( flowLength > 1 ) {
+		const flowClassName = `flow-progress-indicator flow-progress-indicator--${ flowName }`;
```

Comment:
we should probably use the `classnames` library here and do
```
const flowClassName = classnames( 'flow-progress-indicator', `flow-progress-indicator-type-${ flowName }` );
```

Also I think the `--` goes against the coding guidlines. Maybe something like `flow-progress-indicator-type-${ flowName }` might also work